### PR TITLE
Demo for #4732 ignoreMigration issue

### PR DESCRIPTION
**Pull request by MittWillson**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -8,13 +8,13 @@ import (
 // GORM_BRANCH: master
 // TEST_DRIVERS: sqlite, mysql, postgres, sqlserver
 
-func TestGORM(t *testing.T) {
-	user := User{Name: "jinzhu"}
-
-	DB.Create(&user)
+type TableA struct {
```
Comment (longlihale) [inline main_test.go:11]: func (t *TableA) TableName() string {
	return "tableA"
}

File: main_test.go [hunk 2]
```diff
@@ -8,13 +8,13 @@ import (
...
+	Name string `gorm:"size:100;-:migration"`
```
Comment (longlihale) [inline main_test.go:12]: hello, look create table ddl.
```
CREATE TABLE `tableA` ()
```
It is illegal,if you add another field, for example :
```
type TableA struct {
	Name string `gorm:"size:100;-:migration"`
	UserId   int    `gorm:"column:user_id"`
}
```
It works correctly

Comment (MittWillson) [reply main_test.go:12]: Yes, but the issue is the -:migration should only work for alter column, but in here, this column will not be created

Comment (longlihale) [reply main_test.go:12]: `-:migration` means only disable the migration, so this column will not be created. @MittWillson

Comment (MittWillson) [reply main_test.go:12]: @longlihale but if this so, how different between `-` and `-:migration`

Comment (MittWillson) [reply main_test.go:12]: Oh i see

**Pull request closed without merging**